metrics/disttask.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	DispatchingStatus = "dispatching"
+	WaitingStatus     = "waiting"
+	RunningStatus     = "running"
+	CompletedStatus   = "completed"
+)
+
+const (
+	LblTaskStatus  = "status"
+	LblTaskType    = "task_type"
+	LblTaskID      = "task_id"
+	LblSubTaskID   = "subtask_id"
+	LblSchedulerID = "scheduler_id"
+)
+
+var (
+	//DistTaskHistogram      *prometheus.HistogramVec
+	DistTaskDispatcherGauge          *prometheus.GaugeVec
```

Comment:
Prefer to using `DistDDLTaskGauge` instead of `DistTaskDispatcherGauge`. In fact, what we are describing is the status of the task, not the dispatcher.